docker/from-packages/Dockerfile

```
@@ -118,6 +118,10 @@ RUN --mount=type=secret,id=secret_key \
             apt-get -y install code-brand; \
         fi; \
     fi && \
+# Change cool uid/gid to 10000/10001
+    find /opt/cool -user cool -exec chown -h 10000:10001 {} \; && \
```

1001:1001 should be sufficient.

```suggestion
     # Change cool uid/gid to 1001/1001
    find /opt/cool -user cool -exec chown -h 1000:1001 {} \; && \
```

https://stackoverflow.com/a/73359838 seems relevant here. At least I learned a thing or two.